WebRoot/js/zimbraMail/prefs/view/ZmFilterRuleDialog.js

```diff
@@ -203,6 +205,14 @@ function(parent, segment, i) {
 			// TODO: guard against badly specified message
 			select.addOption(formats[i].toPattern(), i == 0, values[i]);
 		};
+
+		var setAriaLabel = function() {
+			select.setAriaLabel(select._selectedValue == ZmFilterRule.GROUP_ANY ? "If any of the following conditions are met" : "If all of the following conditions are met");
```

Comment:
Notes: Will translate newly added text once PR got approved.
`If any of the following conditions are met` , `If all of the following conditions are met`,

Reply:
Added translations.